reference/docs-conceptual/learn/deep-dives/everything-about-hashtable.md

```diff
@@ -814,14 +856,35 @@ Just because you can do something, it doesn't mean that you should. That last on
 bug waiting to happen and would be easily misunderstood by anyone reading your code.
 
 Technically your key doesn't have to be a string but they're easier to think about if you only use
-strings.
+strings. However, indexing doesn't work well with the complex keys.
+
+```powershell
+$ht = @{ @(1,2,3) = "a" }
+$ht
+
+Name                           Value
+----                           -----
+{1, 2, 3}                      a
+```
+
+Accessing a value in the hashtable by its key doesn't always work. For example:
+
+```powershell
+$key = $ht.keys[0]
+$ht.$key
+$ht[$key]
+a
+```
+
+Using the member access (`.`) notation returns nothing. But using the array index (`[]`) notation
```

Comment:
@sdwheeler @DCtheGeek I believe we are actually mistaken here. Please [see my comment here](https://github.com/MicrosoftDocs/PowerShell-Docs/issues/6056#issuecomment-640319093).